web-animations-2/Overview.bs

```diff
@@ -2980,9 +2994,9 @@ enum FillMode { "none", "forwards", "backwards", "both", "auto" };
 :   <code>auto</code>
 ::  Update the description as:
 
-    > Fill backwards and forwards when applied to an
-    > {{GroupEffect}} and no fill when applied to an
-    > {{KeyframeEffect}}.
+    > Fill backwards and forwards when applied to a
+    > {{GroupEffect}} and no fill when applied to a
+    > {{KeyframeEffect!!interface}}.
```

Comment:
Seems like this doesn't actually change what it links to, both https://foremost-electric-knuckle.glitch.me/#the-fillmode-enumeration and https://drafts.csswg.org/web-animations-2/#the-fillmode-enumeration link to
https://drafts.csswg.org/web-animations-1/#keyframeeffect. In general I think less verbosity is better unless we expect to have another definition we need to disambiguate from.

Reply:
I spoke with @tabatkins about this, and he recommended the `!!interface` syntax.